src/table/table.scss

```diff
@@ -18,6 +18,8 @@
 
 @mixin iui-table-header {
   user-select: none;
+  overflow: hidden;
```

Comment:
We could improve the experience for Firefox.

This is what it currently looks like. Notice how the header and body cells are not vertically aligned because the scrollbar takes up space.
![image](https://user-images.githubusercontent.com/9084735/146772172-3b30a5f4-89a5-461d-8ee2-67991ee46591.png)

If we make the scroll track visible on the header, it will line up.
```suggestion
  overflow: hidden;
  @supports not (overflow: overlay) {
    overflow-y: scroll;
  }
```

Would need to also change body to use `overflow: scroll` instead of `overflow: auto`

Reply:
Added